compiler/damlc/tests/daml-test-files/ExceptionUnusedWarning.daml

```diff
@@ -0,0 +1,14 @@
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- @SINCE-LF-FEATURE DAML_EXCEPTIONS
+
+{-# OPTIONS_GHC -fwarn-unused-matches #-}
+
+module ExceptionUnusedWarning where
```

Comment:
this might be a bit confusing without context, since we're checking there's no warning. I'd add a comment saying this is a regression test for https://github.com/digital-asset/daml/issues/11004